spec/system/provider_interface/first_login_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.feature 'See applications' do
+  include CourseOptionHelpers
+  include DfESignInHelpers
+
+  let(:pre_approved_provider_user) do
+    provider = create(:provider, code: 'ABC')
+    create(:provider_user, providers: [provider], dfe_sign_in_uid: nil)
+  end
```

Comment:
FYI: prefer to use instance variables in the integration tests, so we don't clutter the tests with `let` and `before` blocks.